### PR TITLE
added: landingpage structure

### PR DESCRIPTION
**Pull request by ikkurthis1998**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+* {
+	/* font-family: "Lato", sans-serif; */
+	font-family: "Montserrat", sans-serif;
+}
+
+body {
+	margin: 0;
+	padding: 0;
+	background-color: #f2eee4;
```
Comment (jeetsdev) [inline css/index.css:17]: Use variables here.

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+	/* font-family: "Lato", sans-serif; */
+	font-family: "Montserrat", sans-serif;
+}
+
+body {
+	margin: 0;
+	padding: 0;
```
Comment (jeetsdev) [inline css/index.css:16]: It's better to add reset styling properties in the universal selector.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="css/index.css" />
+		<script
+			type="module"
+			src="javascript/index.js"
+			crossorigin="anonymous"
+		></script>
```
Comment (jeetsdev) [inline index.html:12]: Add a js loading attribute like differ to make sure that the script gets executed after the DOM rendering.

**Review by jeetsdev — CHANGES_REQUESTED** [review]

File: css/index.css (hunk 2 above)
Comment (ikkurthis1998) [reply css/index.css:16]: I don't want to set margin and padding zero for all the elements. A personal preference 😅

**Review by samflab — APPROVED** [review]

File: css/index.css (hunk 2 above)
Comment (jeetsdev) [reply css/index.css:14]: It's okay then.

**Review by jeetsdev — APPROVED** [review]
LGTM 💯

**Pull request merged**